<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShuffleRepeatWidget</class>
 <widget class="QWidget" name="ShuffleRepeatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>62</width>
    <height>37</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton, QToolButton:hover, QToolButton:pressed {
	border: 0px;
	background: transparent;
}

#line {
	margin: 3px 0px 3px 0px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="repeat">
     <property name="toolTip">
      <string>Repeat</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/media-playlist-repeat-off.png</normaloff>
       <normalon>:/media-playlist-repeat.png</normalon>:/media-playlist-repeat-off.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="shuffle">
     <property name="toolTip">
      <string>Shuffle</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/media-playlist-shuffle-off.png</normaloff>
       <normalon>:/media-playlist-shuffle.png</normalon>:/media-playlist-shuffle-off.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="action_repeat_off">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Don't repeat</string>
   </property>
  </action>
  <action name="action_repeat_track">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Repeat track</string>
   </property>
  </action>
  <action name="action_repeat_album">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Repeat album</string>
   </property>
  </action>
  <action name="action_repeat_playlist">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Repeat playlist</string>
   </property>
  </action>
  <action name="action_shuffle_off">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Don't shuffle</string>
   </property>
  </action>
  <action name="action_shuffle_album">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shuffle by album</string>
   </property>
  </action>
  <action name="action_shuffle_all">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shuffle all</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
